src/workloads/scale/ScanWithLongLived.yml

```diff
@@ -0,0 +1,78 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
+GlobalDefaults:
+  TrackProportion: &TrackProportion 0
+
+  LongLivedDB: &LongLivedDB longlived # name of the long lived database
+  LongLivedDocumentCount: &LongLivedDocumentCount 1000
+
+  ScannerHotDB: &ScannerHotDB rolling
+
+  LongLivedWrites: &LongLivedWrites 600 per 1 second # x5 threads
+  LongLivedReads: &LongLivedReads 20 per 1 second    # x10 threads
+
+  binomial_10k_int: &binomial_10k_int {^RandomInt: {distribution: binomial, t: 10000, p: 0.5}}
```

Comment:
@ddanderson When you mentioned weighting ids for the long-lived readers/writers, is this what you had in mind?

Reply:
Yes, I think so.  I'm guessing this would produce a roughly 'bell shape curve' centered on 5000?  So the "hot spot" is in the middle of a key space of up to 10000.  I wonder if this (the "t" number) should match the `LongLivedDocumentCount` as well, so the key space gets entirely populated?

And it's quite possible we'll want to adjust both these numbers up - we want to get significant chunks of data to be old and potentially evicted.  Depends on the number of documents, size of documents and number of collections.

Reply:
f87b253

I've adjusted the `t` value down to match the document count.

This PR is just aimed to get something merged that seems reasonable and captures the intent of the technical design. We have WT-6967 to analyze and tweak the values so the key distribution looks the way we expect.